packages/qgrid-core/src/validation/validation.service.d.ts

```diff
@@ -1,6 +1,6 @@
 export declare interface Validator {
-	validate(value: any): boolean;
-	getErrors(): Array<string>;
```

Comment:
@klumba12 check here, please. I updated this type because Validator is using LIVR and this return type described in livr.d.ts.